src/main/java/com/powsybl/openloadflow/network/LfNetwork.java

```diff
@@ -580,6 +597,21 @@ public void fix(boolean minImpedance, double lowImpedanceThreshold) {
     }
 
     private void validateBuses(LoadFlowModel loadFlowModel, ReportNode reportNode) {
+        // DC or AC, if no generator, network is dead
```

Comment:
we might want to add an option to allow computing islands without generators in DC